doc/thing/samples/basic-samples/plate.rst

```diff
@@ -0,0 +1,54 @@
+plate
+=====
+
+Overview
+--------
+
+The plate example simulates a device registering the read car plate in a
+sensor. The purpose of this sample is to show how to register a string value
+as a sensor using the KNoT Zephyr SDK.
+
```

Comment:
Please, make it clear that the value is automatically updated. And how it is updated

Reply:
Ok.